tools/docker_run.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
can you please update the readme on how to you the docker part if it is ready?

Reply:
Yes, I will write some docs on this today

Reply:
+ 1